src/server/graphql/models/Task/taskSchema.js

```diff
@@ -3,27 +3,30 @@ import {
   GraphQLNonNull,
   GraphQLID,
   GraphQLString,
-  GraphQLEnumType
+  GraphQLEnumType,
+  GraphQLFloat
```

Comment:
yep, creating a custom `GraphQLSafeInt` or something similar that has an upper bound of 2^52 -1 just didn't sound appealing.